accepted/PSR-7-http-message-meta.md

```diff
@@ -647,3 +647,46 @@ used to populate the headers of an HTTP message.
 * Anton Serdyuk
 * Phil Sturgeon
 * Chris Wilkinson
+
+## 7. Errata
+
+### 7.1 Validation of Header Names and Values
+
+Some special characters within the name or value of an HTTP header might affect
+the parsing of the serialized message in a way that the contents of unrelated
+headers are changed. This misparsing can open up an application to security
+vulnerabilities. A common type of vulnerability is CRLF injection, allowing
+an attacker to inject additional headers or end the list of headers early.
+
+For this reason classes implementing the `MessageInterface` SHOULD strictly
+validate the header names and contents according to the most recent HTTP
+specification ([RFC 7230#3.2][1] at the time of writing). Incorrect values
+SHOULD be rejected and no attempt SHOULD be made to automatically correct
+the provided values.
```

Comment:
I'm not happy about the “no attempt SHOULD” and like to rephrase that to a real SHOULD NOT somehow.
```suggestion
specification ([RFC 7230#3.2][1] at the time of writing). The implementation
SHOULD reject invalid values and SHOULD NOT make any attempt to automatically correct
the provided values.
```

Reply:
The SHOULD NOT verbiage reads more clearly; nice catch!